regression/cbmc-java/address_space_size_limit2/test.desc

```diff
@@ -0,0 +1,7 @@
+CORE
+Test.class
+--object-bits 8
+^EXIT=10$
+^SIGNAL=0$
+^VERIFICATION FAILED$
```

Comment:
Could we also grep the failure message? This would avoid catching an unrelated error. If the expected behaviour is to NOT see the `too many addressed objects` message, than we should put that in the list of excluded strings.